doc/source/examples/dem/granuheap/granuheap.rst

```diff
@@ -0,0 +1,259 @@
+==================================
+Granuheap
+==================================
+
+This example simulates the flow of a cohesive granular material inside a Granuheap instrument using the discrete element method (DEM). For more detailed information on the concepts and physical meaning of the parameters in Lethe, we strongly recommend visiting `DEM parameters <../../../parameters/dem/dem.html>`_.
```

Comment:
maybe add the link to the granuheap instrument . like granuheap could itself point to the link.